css/app.css

```diff
@@ -1,4 +1,85 @@
+/* http://meyerweb.com/eric/tools/css/reset/
```

Comment:
should remove complied files from repo, because it's a dev version so any dev on run project will compile them